Shared/Samples/Configure electronic navigational charts/ConfigureElectronicNavigationalChartsView.swift

```diff
@@ -0,0 +1,283 @@
+                    VStack(alignment: .leading) {
+                        Text(encFeature.acronym)
+                        Text(encFeature.description)
+                    }
+                    .padding(5)
+                }
+                .onSingleTapGesture { screenPoint, _ in
+                    tapPoint = screenPoint
+                }
+                .task(id: tapPoint) {
+                    // Identifies and selects a tapped feature.
+                    guard let tapPoint else {
+                        return
+                    }
+                    
+                    do {
+                        try await selectENCFeature(screenPoint: tapPoint, proxy: mapViewProxy)
+                    } catch {
+                        self.error = error
+                    }
+                }
+        }
+        .toolbar {
+            ToolbarItem(placement: .bottomBar) {
+                Button("Display Settings") {
+                    isShowingDisplaySettings = true
+                }
+                .popover(isPresented: $isShowingDisplaySettings) {
+                    ENCDisplaySettingsView()
+                        .presentationDetents([.fraction(0.5)])
+                        .frame(idealWidth: 320, idealHeight: 280)
+                }
+            }
+        }
+        .task {
+            // Sets up the sample when it opens.
+            do {
+                try await model.addENCExchangeSet()
+                model.configureENCDisplaySettings()
+            } catch {
+                self.error = error
+            }
+        }
+        .errorAlert(presentingError: $error)
+    }
+    
+    /// Selects an ENC feature identified at a screen point.
+    /// - Parameters:
+    ///   - screenPoint: The screen coordinate of the geo view at which to identify.
+    ///   - proxy: The map view proxy used to identify the screen point.
+    private func selectENCFeature(screenPoint: CGPoint, proxy: MapViewProxy) async throws {
+        model.encLayer?.clearSelection()
+        
+        // Uses the proxy to identify the layers at the screen point.
+        let identifyResults = try await proxy.identifyLayers(
+            screenPoint: screenPoint,
+            tolerance: 10
+        )
+        
+        // Gets the ENC layer and feature from the identify results.
+        guard let result = identifyResults.first(where: { $0.layerContent is ENCLayer }),
+              let encLayer = result.layerContent as? ENCLayer,
+              let encFeature = result.geoElements.first as? ENCFeature else {
+            return
+        }
+        
+        // Selects the feature using the layer.
+        encLayer.select(encFeature)
+        model.encLayer = encLayer
+        
+        // Sets the callout to display on the feature.
+        let tapLocation = proxy.location(fromScreenPoint: screenPoint)
+        calloutPlacement = .geoElement(encFeature, tapLocation: tapLocation)
```

Comment:
When I tap out of the enc layer area, the callout won't be dismissed. Maybe `calloutPlacement = nil` on around line 90.